### PR TITLE
Update mountinfo to support FreeBSD

### PR DESCRIPTION
**Pull request by Red54**
This fixes https://github.com/containerd/nerdctl/issues/868

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -122,7 +122,7 @@ require (
 	github.com/mitchellh/go-homedir v1.1.0 // indirect
 	github.com/mitchellh/mapstructure v1.4.3 // indirect
 	github.com/moby/locker v1.0.1 // indirect
-	github.com/moby/sys/mountinfo v0.6.1 // indirect
+	github.com/moby/sys/mountinfo v0.6.2-0.20220518012833-28ef7ed18e6a // indirect
```
Comment (AkihiroSuda) [inline go.mod:125]: Requesting the v0.6.2 release: https://github.com/moby/sys/pull/114#issuecomment-1129849702

**Comment by akhramov** [comment]
Now that the new moby/sys version with @Red54's code is released, we can bump it.

I created #1093 for this, which also re-enables CI, fixes docs, etc. I hope you don't mind @Red54 and thank you so much for the fix!

**Comment by akhramov** [comment]
Now that the new moby/sys version with @Red54's code is released, we can bump it.

I created #1093 for this, which also re-enables CI, fixes docs, etc. I hope you don't mind @Red54 and thank you so much for the fix!

**Comment by AkihiroSuda** [comment]
Thanks, let me close this in favor of #1093

**Pull request closed without merging**